l91/webpack-encore/1.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "copy-from-recipe": {
+        "assets/": "assets/",
```

Comment:
You could copy the assets from the package.

Reply:
Thx for the review! I'm not sure what you mean with this. Can you explain?

Reply:
Instead of adding this code in the recipe, you may add it to your package instead. That will make it easier for you to make updates if any.

Reply:
oh thats great didn't know symfony/flex allow to access the vendor directory. So I can go here with:

```bash
        "vendor/l91/webpack-encore/assets/": "assets/",
```

directory or how to reference the package directory here?

Reply:
Have a look at this package, it is a good example.  
https://github.com/symfony/recipes-contrib/blob/master/synolia/sylius-scheduler-command-plugin/1.0/manifest.json
https://github.com/synolia/SyliusSchedulerCommandPlugin

Reply:
That looks great. Thx will refractor it and ping you again when its finished.